draft-ietf-quic-multipath.md

```diff
@@ -994,11 +994,10 @@ closed after the idle timeout expires.
 
 # New Frames {#frames}
 
-All the new frames MUST only be sent in 1-RTT packet, and MUST NOT
-use other encryption levels.
+All the new frames MUST only be sent in 1-RTT packet.
 
-If an endpoint receives multipath-specific frames from packets of
-other encryption levels, it MUST return MP_PROTOCOL_VIOLATION
+If an endpoint receives multipath-specific frames from other
+types of packets, it MUST return MP_PROTOCOL_VIOLATION
 as a connection error and close the connection.
```

Comment:
Unrelated to this PR, but this is not an RPC, and as such, "returning an error" might not be the best term to use.
```suggestion
If an endpoint receives a multipath-specific frame in a different packet type,
it MUST close the connection with an error of type MP_PROTOCOL_VIOLATION.
```